src/Tribe/Views/V2/View.php

```diff
@@ -1138,6 +1138,7 @@ protected function filter_template_vars( array $template_vars ) {
 	 * @return array The arguments, ready to be set on the View repository instance.
 	 */
 	protected function setup_repository_args( Context $context = null ) {
+		bdump('view_setup_repository_args');
```

Comment:
* :no_entry_sign: Expected 1 spaces after opening parenthesis; 0 found
* :no_entry_sign: Expected 1 spaces before closing parenthesis; 0 found
* :no_entry_sign: There must be a single space after the first parenthesis
* :no_entry_sign: There must be a single space before the last parenthesis

<sub>hash: b906ba7ed9f89ef08b4d4db0374c381c</sub>